backend/benefit/applications/templates/style.html

```diff
@@ -0,0 +1,163 @@
+<style>
```

Comment:
A looot of CSS for a backend :D I guess there's no other way.

Reply:
They also have to be in `html` file because PDFKit has a known issue on crashing for any local content.